src/platform_impl/android/ndk_glue.rs

```diff
@@ -188,6 +210,7 @@ lazy_static::lazy_static! {
   pub(crate) static ref CONTEXTS: Mutex<BTreeMap<ActivityId, AndroidContext>> = Mutex::new(Default::default());
   static ref WINDOW_MANAGER: Mutex<BTreeMap<ActivityId, GlobalRef>> = Mutex::new(Default::default());
   pub(crate) static ref ACTIVITY_CREATED_SENDERS: Mutex<BTreeMap<ActivityId, Sender<()>>> = Mutex::new(Default::default());
+  static ref INTENT_URLS: Mutex<Vec<url::Url>> = Mutex::new(Default::default());
```

Comment:
Using global mutable state for intent URLs. How does this interact with multi-window support? What if multiple windows exist? How does the intent get delivered to the correct window?

Reply:
the intent URL is not delivered to a particular window - it is instead sent as a global event Event::Opened
i wanted to include the URLs in the android Event enum but I can't since it has to implement Copy - that's why I'm using this global variable.